src/components/custom-tom-select.scss

```diff
@@ -187,18 +187,6 @@
   @extend .form__dropdown;
 
   height: 100% !important;
-
-  // border: 1px solid #d0d0d0;
-  // padding: 8px 8px;
-  // width: 100%;
-  // overflow: hidden;
-  // position: relative;
-  // z-index: 1;
-  // box-sizing: border-box;
-  // box-shadow: none;
-  // border-radius: 3px;
-  // display: flex;
-  // flex-wrap: wrap;
 }
 .ts-wrapper.multi.has-items .ts-control {
   padding: calc(8px - 2px) 8px calc(8px - 2px - 3px);
```

Comment:
@Jeremy-Walton Can we use our tokens here instead of the static styles for padding. Some padding voodoo happening with this one.